sqlglot/transforms.py

```diff
@@ -348,7 +348,10 @@ def _udtf_type(u: exp.Unnest, has_multi_expr: bool) -> t.Type[exp.Func]:
                 )
             )
 
-        for join in expression.args.get("joins") or []:
+        joins = expression.args.get("joins") or []
+        for join in list(
+            joins
+        ):  # support safe removal by iterating over copy list for multiple joins
```

Comment:
```suggestion
        for join in list(joins):
```

Reply:
Commited: [27c7753](https://github.com/tobymao/sqlglot/pull/4364/commits/27c775376c60b20ca4679d9c1095a776a3ac1a2d)

Reply:
This is a different comment, the formatting here can be inlined. The explanation might be good to have as a reminder but imo it's a common Python pattern so we can remove it too

Reply:
Suggestion accepted.